apps/namada-interface/src/App/Staking/Staking.tsx

```diff
@@ -68,9 +99,11 @@ export const Staking = (props: Props): JSX.Element => {
   } = props;
 
   // these 2 are needed for validator details
-  const stakingPositionsWithSelectedValidator = myStakingPositions.filter(
-    (validator) => validator.validatorId === selectedValidatorId
-  );
+  const stakingPositionsWithSelectedValidator =
+    myStakingPositions &&
```

Comment:
`myStakingPosition` is of type `MyStakingPosition[]`, so either `myStakingPosition` is always true or the type has to change. Kinda weird that typescript does not show it as "always true"

Reply:
Odd that it was there, but now I removed it and TS seems to be happy about it as expected.